samples/dynamically_add_module_sample/README.md

```diff
@@ -57,11 +55,11 @@ Windows
 
 ##How to run the sample
 
-Linux
+#Linux
 
-<TODO: Put here a description on how to run this sample> 
+./build/sample/dynamically_add_module_sample/dynamically_add_module_sample sample/dynamically_add_module_sample/src/modules_lin.json sample/dynamically_add_module_sample/src/links_lin.json
```

Comment:
Surround in backticks (`) so it looks like something you type at the terminal. Same for the Windows command below.